examples/EmbedExternalUI/EmbedExternalExampleUI.cpp

```diff
@@ -271,6 +269,9 @@ class EmbedExternalExampleUI : public UI
         UI::sizeChanged(width, height);
 
 #if defined(DISTRHO_OS_MAC)
+        NSRect rect = fView.frame;
```

Comment:
does this even do anything? you are not calling NSView functions which could trigger callbacks..

Reply:
This is actually a call behind the scenes: 

` fView.frame = rect;`

It translates to

`[fView setFrame:rect];`

It is syntactic sugar, ObjC calls these "properties".

Reply:
ah I was not aware of this, thanks. ObjC is quite weird..

Reply:
Apple gradually started adding sugar after iOS was invented, if I correctly recall. The core OOP ideas come from Smalltalk, hence the weirdness...